StudCity.Frontend/src/pages/Profile/EditProfile/index.jsx

```diff
@@ -0,0 +1,173 @@
+
+import React, { useState } from "react";
+import Avatar from "../../../UI/Avatar";
+import Button from "../../../UI/Button";
+import TextField from "../../../UI/fields/TextField";
+import SelectField from "../../../UI/fields/SelectField";
+import ValidateTextField from "../../../UI/fields/ValidateTextField";
+import UploadAvatar from "../../RegistrationComplete/steps/UploadAvatar";
+import { 
+	SELECT_BACKGROUND_URLS,
+	DEFAULT_BACKGROUND_URL,
+ } from "../../../utils/constants";
+//import UploadImageInput from "../../../UI/fields/UploadImageInput";
+import DatePicker from "../../../UI/DatePicker";
+import CustomDialog from "../../../UI/CustomDialog";
+
+function EditProfile() {
+	const styleForButton = "mt-auto text-base w-16 h-10 mt-auto mx-0";
+	const containerWithInputs  = "flex justify-between w-full mt-10 items-center";
+	const inputBlock = "w-1/2 flex justify-between items-center";
+	const spanWithInputsBlocks = "ml-16 text-left text-base";
+	const textFieldStyle = "w-60 h-10";
+	const [avatar, setAvatar] = useState("");
+	const [backgroundImage, setBackgroundImage] = useState("");
+	const [showDialogForAvatar, setShowDialogForAvatar] = useState(false);
+	const [showDialogForBackground, setShowDialogForBackground] = useState(false);
+	return(
+		<div className="w-full h-fit flex flex-col">
+			{showDialogForAvatar 
+				? 
+				<CustomDialog close={setShowDialogForAvatar}>
+					<UploadAvatar 
+						className={"w-[800px] h-[600px]"} 
+						setAvatar={setAvatar}
+					/>
+				</CustomDialog> 
+				: 
+				<></> 
+			}
+			{showDialogForBackground 
+				? 
+				<CustomDialog close={setShowDialogForBackground}>
+					<div className="w-[1000px] h-[600px] flex ">
+						<div className="w-4/5 h-[90%] overflow-scroll grid grid-cols-2 m-auto gap-10">
+							{
+								SELECT_BACKGROUND_URLS.map(( element, index ) => (
+									<div className="rounded-tl-[60px] overflow-hidden w-full  h-56" key = {index}>
+										<img 
+											src={element} 
+											alt="" 
+											className="w-full h-full object-cover"	
+											onClick={() => setBackgroundImage(element)}  
+										/>
+									</div>
+								)
+								)
+							}
+						</div>
+					</div>
+				</CustomDialog> 
+				: 
+				<> </> 
+			}
+			<div className="w-10/12 h-full flex flex-col mx-auto pb-11">
+				<div className="h-24 w-full  border-b-2 border-[#D1D7D4] flex items-center">
+					<h4 className="text-3xl font-medium" >Settings</h4>
+				</div>
+				<div className="h-5/6 flex w-full">
+					<div className="w-3/12 h-full "></div>
+					<div className="w-9/12 h-full flex flex-col">
+						<div className="h-52 w-full overflow-hidden rounded-tl-[60px] mt-6 shadow-form relative">
+							<img 
+								src={backgroundImage === "" ? DEFAULT_BACKGROUND_URL : backgroundImage} 
```

Comment:
better use just backgroundImage and set DEFAULT_BACKGROUND_URL directly into state as default